net/https-dns-proxy/Makefile

```diff
@@ -1,15 +1,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=https_dns_proxy
-PKG_VERSION:=2018-04-23
+PKG_VERSION:=2019-07-21
 PKG_RELEASE=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_MIRROR_HASH:=24b7e4238c37e646f33eee3a374f6b7beb5c167b9c5008cc13b51e5f1f3a44ea
+PKG_MIRROR_HASH:=57e9c655622772fdc4031210fd30bb36faa0a0a253aeb31aac1129c641f07b7f
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
 PKG_SOURCE_URL:=https://github.com/aarond10/https_dns_proxy/
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=bea68401330e611f6e9b75cec84e2dc4e81e52de
+PKG_SOURCE_VERSION:=4e47de7b27bffe153670b7c1c0c29776ea1d6cd6
 PKG_MAINTAINER:=Aaron Drew <aarond10@gmail.com>
 PKG_LICENSE:=MIT
```

Comment:
Add PKG_LICENSE_FILES and rearrange the whole block to be consistent with other packages. For example: https://github.com/neheb/packages/blob/bab985450083e4071d06f1fa26ffa37544a146db/libs/fbthrift/Makefile